src/energyplus/ForwardTranslator/ForwardTranslateZoneHVACTerminalUnitVariableRefrigerantFlow.cpp

```diff
@@ -413,6 +415,18 @@ namespace energyplus {
     idfObject.setDouble(ZoneHVAC_TerminalUnit_VariableRefrigerantFlowFields::MaximumOutdoorDryBulbTemperatureforSupplementalHeaterOperation,
                         modelObject.maximumOutdoorDryBulbTemperatureforSupplementalHeaterOperation());
 
+    // ControllingZoneorThermostatLocation
+    if (boost::optional<ThermalZone> tz = modelObject.controllingZoneorThermostatLocation()) {
+      idfObject.setString(ZoneHVAC_TerminalUnit_VariableRefrigerantFlowFields::ControllingZoneorThermostatLocation, tz->name().get());
+    } else if (auto airLoop = modelObject.airLoopHVAC()) {
+      auto zones = airLoop->thermalZones();
+      if (zones.size() == 1u) {
+        if (auto zone = translateAndMapModelObject(zones.front())) {
+          idfObject.setString(ZoneHVAC_TerminalUnit_VariableRefrigerantFlowFields::ControllingZoneorThermostatLocation, zone->name().get());
+        }
+      }
+    }
```

Comment:
Write the ControllingZoneorThermostatLocation: if supplied, use that. Otherwise, if it's on a main AirLoopHVAC branch, pick up the first thermal zone that is served by the AirLoopĤVAC